gen/calc.py

```diff
@@ -557,7 +557,8 @@ def calculate_set(parameter):
         'minuteman_forward_metrics': 'false',
         'minuteman_min_named_ip_erltuple': calculate_minuteman_min_named_ip_erltuple,
         'minuteman_max_named_ip_erltuple': calculate_minuteman_max_named_ip_erltuple,
-        'mesos_isolation': calculate_mesos_isolation,
+        'mesos_isolation': 'cgroups/cpu,cgroups/mem,disk/du,network/cni,filesystem/linux,docker/runtime,docker/volume'
```

Comment:
I updated the isolation scheme from the method to append the metrics isolation in the returned list. This was modified when pods was merged last week, which is where the rebase broke down for me earlier this week.